src/status_im/constants.cljs

```diff
@@ -25,6 +25,13 @@
 
 (def default-network "testnet_rpc")
 
+(def default-wallet-transactions
```

Comment:
looks like this is not a constant, because `:checked?` can be changed

Reply:
This is the default used when loading the app. Every restart loads this default. 
Can you think about another way to do this?